test/BUILD.gn

```diff
@@ -230,6 +230,8 @@ test("brave_unit_tests") {
     sources += [
       "../utility/importer/chrome_importer_unittest.cc",
       "//brave/app/brave_command_line_helper_unittest.cc",
+      "//brave/browser/extensions/chrome_content_verifier_delegate_unittest.cc",
+      "//brave/browser/extensions/install_verifier_unittest.cc",
```

Comment:
I believe these need to go under the `enable_extensions` guard below where the dep was added.

Reply:
Checking this out now...

Reply:
Good catch! 😄

Reply:
Fixed 😄